assets/src/components/lateView.tsx

```diff
@@ -5,11 +5,9 @@ import { VehiclesByRouteIdContext } from "../contexts/vehiclesByRouteIdContext"
 import { StateDispatchContext } from "../contexts/stateDispatchContext"
 import {
   bangIcon,
-  busBangIcon,
   lateViewGhostIcon,
   lateViewGhostWithWaiverIcon,
   upRightIcon,
-  userXIcon,
```

Comment:
*question:* were these the only uses of these SVGs? If so, can we remove them from the codebase?

Reply:
I believe so, and agreed, if we need them later we can always pull them out of the git history.